docs/source/index.rst

```diff
@@ -25,6 +25,78 @@ then!
 .. literalinclude:: trustme-trio-example.py
 
 
+CLI reference
+=============
+
+**All options:**
+
+.. code-block:: console
+
+  $ python -m trustme --help
+  usage: trustme [-h] [-d DIR] [-i [IDENTITIES [IDENTITIES ...]]]
+                [--common-name COMMON_NAME] [--key-size KEY_SIZE] [-q]
+
+  optional arguments:
+    -h, --help            show this help message and exit
+    -d DIR, --dir DIR     Directory where certificates and keys are written to.
+                          Defaults to cwd.
+    -i [IDENTITIES [IDENTITIES ...]], --identities [IDENTITIES [IDENTITIES ...]]
+                          Identities for the certificate. Defaults to 'localhost
+                          127.0.0.1 ::1'.
+    --common-name COMMON_NAME
+                          Also sets the deprecated 'commonName' field (only for
+                          the first identity passed).
+    --key-size KEY_SIZE   Key size of the certificate generated. Defaults to
+                          2048.
+    -q, --quiet           Doesn't print out helpful information for humans.
+
+**Default configuration:**
+
+.. code-block:: console
+
+  $ cd /tmp/
+  $ python -m trustme
+  Generated a certificate for 'localhost', '127.0.0.1', '::1'
+  Configure your server to use the following files:
+    cert=/tmp/server.pem
+    key=/tmp/server.key
+  Configure your client to use the following files:
+    cert=/tmp/client.pem
+
+**Designate different identities:**
+
+.. code-block:: console
+
+  $ python -m trustme -i www.example.org example.org
+  Generated a certificate for 'www.example.org', 'example.org'
+  Configure your server to use the following files:
+    cert=/tmp/server.pem
+    key=/tmp/server.key
+  Configure your client to use the following files:
+    cert=/tmp/client.pem
+
+**Generate files into a directory:**
+
+.. code-block:: console
+
+  $ mkdir /tmp/a
+  $ python -m trustme -d /tmp/a
+  Generated a certificate for 'localhost', '127.0.0.1', '::1'
+  Configure your server to use the following files:
+    cert=/tmp/a/server.pem
+    key=/tmp/a/server.key
+  Configure your client to use the following files:
+    cert=/tmp/a/client.pem
+
+**Configure certs for server/client:**
+
+.. code-block:: console
+
+  $ gunicorn --keyfile /tmp/a/server.key --certfile/tmp/a/server.cert app:app
```

Comment:
Should we show a minimal app.py? It could just be the Flask Hello World. If we had that, then this doc would allow users to have something working at the end. Integrating trustme in a given codebase can be challenging to debug, and having a working example helps a lot.